src/EnergyPlus/PlantLoopHeatPumpEIR.cc

```diff
@@ -341,44 +559,60 @@ void EIRPlantLoopHeatPump::doPhysics(EnergyPlusData &state, Real64 currentLoad)
         eirModifierFuncPLR = 0.0;
     }
 
-    this->powerUsage = (this->loadSideHeatTransfer / this->referenceCOP) * eirModifierFuncPLR * eirModifierFuncTemp;
+    this->powerUsage =
+        (availableCapacity / this->referenceCOP) * eirModifierFuncPLR * eirModifierFuncTemp * InputPowerMultiplier * this->cyclingRatio;
```

Comment:
the above was corrected to include cycling ratio (= 1 above min PLR)

Reply:
@mjwitte here is the change that accompnies the change to example file EIRfPLR curves. The old equation gave full load power because eirModifierFuncPLR = 1. Now power is proportional to PLR and adds in a factor where below minPLR the HP is cycling on and off (cyclingRatio = 1 above minPLR). So I guess I didn't really change the equation, but corrected the input file curve.

Reply:
OK, I remember this now, the old equation used this->loadSideHeatTransfer, instead of availableCapacity, which did account for PLR. The only transition I can think of is updating a unity curve.

Reply:
This adds to the age-old confusion of what an EIRfPLR curve should look like. If efficiency is flat vs plr should the curve be 1.0 constant (no modification) or should it be a 45-degree line? If you use `this->loadSideHeatTransfer` which I find more intuitive, then it should be a 1.0 constant curve for flat efficiency. What does the EIR chiller use?

Reply:
The EIR chiller uses:

    this->Power = (AvailChillerCap / ReferenceCOP) * this->ChillerEIRFPLR * this->ChillerEIRFT * FRAC;

Reply:
Maybe I'm the only one who's old-age confused. The recently added HeatPump:AirToWater:FuelFired:Heating was based on this heat pump, so it's using the other method:
`2224: this->fuelRate = this->loadSideHeatTransfer / (this->referenceCOP * CRF) * eirModifierFuncPLR * eirModifierFuncTemp * eirDefrost;`
@jcyuan2020 Noting this here. Not sure if we need to change HeatPump:AirToWater:FuelFired:Heating or just make sure documentation is very clear and possibly add some checks/warning to catch eirfplr curves that are assuming the wrong basis.

Reply:
@mjwitte Noted. 

This would definitely make some impacts the way how the heating/cooling energy is calculated for the plant-loop heat pumps or chillers---and it is likely that the users would find this difference in the results between the old and new versions soon. A previous discussion about this issue can be also found in PR #9792 (addressing earlier issue post #9252) about the differences of the two methods.